src/Persistence/Csv.php

```diff
@@ -258,7 +258,7 @@ public function tryLoadAny(Model $model): ?array
         }
 
         $data = $this->typecastLoadRow($model, $data);
-        $data['id'] = $this->line;
+        $data[$model->id_field] = $this->line;
```

Comment:
this was a bug I belive

Reply:
Good catch!

Reply:
@PhilippGrashoff can you approve this PR?